zerver/worker/queue_processors.py

```diff
@@ -1153,6 +1154,15 @@ def failure_processor(event: Dict[str, Any]) -> None:
             )
             user_profile = get_user_profile_by_id(event["user_profile_id"])
             reactivate_user_if_soft_deactivated(user_profile)
+        elif event["type"] == "scrub_deactivated_realms":
+            realms_to_scrub = Realm.objects.filter(
+                deactivated=True,
+                scheduled_deletion_date__lte=timezone_now(),
+                scheduled_deletion_date__gt=timezone_now() - datetime.timedelta(days=1),
```

Comment:
We should just limit to `scheduled_deletion_date__lte=timezone_now()` -- if the cron job gets skipped for any reason, then scrubbing the deactivated realms should scrub _all_ of the ones past the deletion date.